tests/test_generate.py

```diff
@@ -308,7 +354,8 @@ def test_generate(yaml_dir, tmpdir):
 
     # Upload new blob, build only latest, and verify only latest is updated.
     new_metadata = "docs.metadata.newer"
-    latest_html_blob_name = "go-cloud.google.com/go/storage-v1.40.1.tar.gz"
+    _add_uuid_to_metadata(yaml_dir, new_metadata)
```

Comment:
It feels like we should do this in `api_dir`, next to where `docs.metadata` gets updated.